aws-api-appsync/src/main/java/com/amplifyframework/api/aws/TemporalDeserializers.java

```diff
@@ -30,53 +30,69 @@
  * AppSync temporal scalars</a>.
  */
 public final class TemporalDeserializers {
+
     /**
      * Deserializer of Temporal.Date, an extended ISO-8601 Date string, with an optional timezone offset.
-     *
+     * <p>
      * Based on the <a href=https://docs.aws.amazon.com/appsync/latest/devguide/scalars.html> AWS AppSync AWSDate
      * scalar.</a>
      */
     public static final class DateDeserializer implements JsonDeserializer<Temporal.Date> {
         @Override
-        public Temporal.Date deserialize(JsonElement json, Type typeOfT, JsonDeserializationContext context)
-                throws JsonParseException {
```

Comment:
I believe `throws JsonParseException` needs to stay here, since this method can still throw a JsonParseException?